styleguide/components/41-organisms/timeline/timeline.twig

```diff
@@ -11,17 +11,17 @@
             {% include '@image-gallery--multiple' with {
               "items": [
                 {
-                  "src": "https://loremflickr.com/704/440/ghent?random=1",
+                  "src": "https://loremflickr.com/800/500/ghent?random=1",
```

Comment:
Dit is een organisms, best geen content voorzien: https://via.placeholder.com/800x500&text=8:5+(800x500)

Reply:
Aangepast.